voyages-ms/app-deploy.yaml

```diff
@@ -2,43 +2,43 @@ apiVersion: appsody.dev/v1beta1
+    commit.image.appsody.dev/date: Wed Jul 1 14:14:18 2020 -0500
+    commit.image.appsody.dev/message: Created docs-archive and removed mkdocs.yml
+      (#56)
     commit.stack.appsody.dev/contextDir: /incubator/nodejs-express
-    commit.stack.appsody.dev/date: Wed Jun 10 15:34:28 2020 +0100
-    commit.stack.appsody.dev/message: 'meta: Replace the maintainer info for node
-      express stacks (#829)'
-    image.opencontainers.org/created: "2020-06-19T16:12:57+01:00"
-    image.opencontainers.org/documentation: https://github.com/djones6/refarch-kc-ms
-    image.opencontainers.org/revision: c5d07b3c46b576e6d02ed693ec49e4d3df77e069
-    image.opencontainers.org/source: |
-      https://github.com/djones6/refarch-kc-ms/tree/update-helm
-    image.opencontainers.org/url: https://github.com/djones6/refarch-kc-ms
+    commit.stack.appsody.dev/date: Wed Jul 1 16:05:26 2020 +0100
+    commit.stack.appsody.dev/message: 'nodejs-express: Install ca-certificates when
+      building production image (#838)'
+    image.opencontainers.org/created: "2020-07-09T10:08:47+02:00"
+    image.opencontainers.org/documentation: https://github.com/jesusmah/refarch-kc-ms
+    image.opencontainers.org/revision: 8ed8440988592ef51c772cdbece4f4bd411014ba-modified
+    image.opencontainers.org/source: https://github.com/jesusmah/refarch-kc-ms/tree/master
+    image.opencontainers.org/url: https://github.com/jesusmah/refarch-kc-ms
     stack.appsody.dev/authors: Gireesh Punathil <gireeshpunathil>
     stack.appsody.dev/configured: docker.io/appsody/nodejs-express:0.4
-    stack.appsody.dev/created: "2020-06-10T14:37:32Z"
+    stack.appsody.dev/created: "2020-07-01T15:09:37Z"
     stack.appsody.dev/description: Express web framework for Node.js
     stack.appsody.dev/digest: sha256:10ef27f63ce277b69666f8a12723d223edc625492cdf95df96232ebbccaab3bb
     stack.appsody.dev/documentation: https://github.com/appsody/stacks/tree/master/incubator/nodejs-express/README.md
     stack.appsody.dev/licenses: Apache-2.0
-    stack.appsody.dev/revision: 41e6af7f5aa970da76cd1b2b10b17ee088d43cb4
+    stack.appsody.dev/revision: 06ea2933df2b6f831fef31ddcbe24c7e0135170c
     stack.appsody.dev/source: https://github.com/appsody/stacks/tree/master/incubator/nodejs-express/image
-    stack.appsody.dev/tag: docker.io/appsody/nodejs-express:0.4.11
+    stack.appsody.dev/tag: docker.io/appsody/nodejs-express:0.4.13
     stack.appsody.dev/title: Node.js Express
     stack.appsody.dev/url: https://github.com/appsody/stacks/tree/master/incubator/nodejs-express
   creationTimestamp: null
   labels:
+    app.appsody.dev/name: refarch-kc
     app.kubernetes.io/part-of: refarch-kc
     image.opencontainers.org/title: voyages-ms
     stack.appsody.dev/id: nodejs-express
-    stack.appsody.dev/version: 0.4.11
+    stack.appsody.dev/version: 0.4.13
   name: voyages-ms
 spec:
-  applicationImage: ibmcase/kcontainer-voyages-ms:0.1.27
+  applicationImage: ibmcase/kcontainer-voyages-ms:test
```

Comment:
Change to `latest` here and I think we're good.

Reply:
The image tag in master is now `latest` (as of https://github.com/ibm-cloud-architecture/refarch-kc-ms/commit/6343e775dea16b9a4e9bb7fb8ea0e57d668f58a5), so when resolving conflicts this PR can just take the latest version and change the order topic name.